src/modules/Builder/features/ActivityItems/ItemConfiguration/GroupedSelectSearchController/ItemTypeTooltip/TooltipComponents/TooltipComponents.utils.test.ts

```diff
@@ -0,0 +1,34 @@
+import { getSelectionSvgId, getSelectionPerRowSvgId } from './TooltipComponents.utils';
+
+describe('getSelectionSvgId', () => {
+  test.each([
+    [1, true, 'radio-button-outline'],
+    [2, true, 'radio-button-empty-outline'],
+    [2, false, 'checkbox-empty-outline'],
+    [3, false, 'checkbox-filled'],
+  ])(
+    'returns correct svg id for index %i and isSingleSelection %s',
+    (index, isSingleSelection, expected) => {
+      const result = getSelectionSvgId(index, isSingleSelection);
+      expect(result).toEqual(expected);
+    },
+  );
+});
+
+describe('getSelectionPerRowSvgId', () => {
```

Comment:
Could please add a few assertions for the case when it returns `radio-button-empty-outline` and `checkbox-empty-outline`